roles/signals.py

```diff
@@ -20,3 +20,24 @@ def my_callback(sender, **kwargs):
     # pylint: disable=protected-access
     if sender.name == Repository._meta.app_label:
         sync_groups_permissions()
+
+
+@receiver(post_save)
+def add_creator_permission(sender, **kwargs):
+    """
+    Give the creator of a repository admin permissions
+    over that repository.
+    """
+    from roles.api import assign_user_to_repo_group, roles_init_new_repo
+    from roles.permissions import GroupTypes
+    instance = kwargs.pop("instance", None)
+    if not kwargs["created"]:
+        return
+    if instance.__class__.__name__ != "Repository":
```

Comment:
Since we're importing `Repository` anyway can't we compare this against `Repository.__name__` instead?